LICENSES/BSD-2-Clause.txt

```diff
@@ -0,0 +1,22 @@
+Copyright (c) <year> <owner>. All rights reserved.
```

Comment:
Shouldn't there be a year given?

Reply:
From my understanding that's the license text and year plus text come from the individual files.